pkg/agent/controller/traceflow/packetin.go

```diff
@@ -288,7 +288,7 @@ func getCTDstValue(matchers *ofctrl.Matchers, isIPv6 bool) (string, error) {
 
 func getNetworkPolicyObservation(tableID uint8, ingress bool) *opsv1alpha1.Observation {
 	ob := new(opsv1alpha1.Observation)
-	ob.Component = opsv1alpha1.NetworkPolicy
+	ob.Component = opsv1alpha1.NetworkPolicy_
```

Comment:
Because of merging CRDs to crd.antrea.io, the const variable NetworkPolicy is duplicated with struct NetworkPolicy, I renamed the const variable, but I don't come up with a suitable name.

Reply:
The variable name can start with their type, for example `ComponentNetworkPolicy`